backend/migrations/2023-03-10-004745_game_challenge/up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE game_challenges (
+    id serial primary key not null,
```

Comment:
Did you try the `identity` instead of serial https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-identity-column/